app/editor/src/features/content/tool-bar/sections/filter/ShowOnlySection.tsx

```diff
@@ -29,7 +29,6 @@ export interface IShowOnlySectionProps {}
  */
 export const ShowOnlySection: React.FC<IShowOnlySectionProps> = () => {
```

Comment:
Fixed a number of bugs with this component and state when selecting or deselecting multiple options.